.circleci/config.yml

```diff
@@ -46,13 +77,20 @@ jobs:
             # Get the longuest line width (ignoring release links)
             test $(cat CHANGELOG.md | grep -Ev "^\[.*\]: https://github.com/openfun" | wc -L) -le 80
 
+  # ---- Docker jobs ----
   # Build the Docker image used in development
   build-docker:
     docker:
-      - image: circleci/buildpack-deps:stretch
+      - image: cimg/base:current
+        auth:
+          username: $DOCKER_HUB_USER
+          password: $DOCKER_HUB_PASSWORD
     working_directory: ~/fun
     steps:
+      # Checkout repository sources
       - checkout
+      - *docker-login
```

Comment:
Not sure this is required here.

Reply:
Because it is a development image?

Reply:
Because the only stage it is required is to push build image to dockerhub. In all other jobs, we pull image with our credentials using:

```yaml
        auth:
          username: $DOCKER_HUB_USER
          password: $DOCKER_HUB_PASSWORD
```